api/.docker/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "credsStore": "ecr-login"
```

Comment:
Created this file to resolve this warning:

![image](https://github.com/user-attachments/assets/6f6b1eb4-b787-444f-b80e-13907611f979)